src/server/generators/title.js

```diff
@@ -8,6 +8,9 @@
 export default function titleGenerator (appId, { attribute } = {}, type, data) {
   return {
     text () {
+      if (data === undefined || data === null || data === false) {
```

Comment:
I think your suggestion is ok if we don't want any `<title>` tag when `metaInfo.title` is set to `0` or `''`. Do you agree with this behaviour?
However, some people could expect to get `<title>0</title>` and `<title></title>` respectively.
What do you think? :wink:

Reply:
It doesnt sound like a valid / real-world use-case to me. Html-wise there is no use for setting an empty title tag, so that would leave `0`. But if somebody really needs they should just set `'0'` which would still work :)

Reply:
Totally agree :smile: I just updated the function.